NAMESPACE

```diff
@@ -8,12 +8,14 @@ export(failure)
 export(is_binary_vector)
 export(is_data_frame)
 export(is_decreasing_vector)
+export(is_diagonal_matrix)
 export(is_existing_dir)
```

Comment:
It's annoying that tests don't fail if the entity is not exported in the namespace. Welcome to R

Reply:
How can the tests know if you want to export it, unless you include it in the NAMESPACE export() ?

Reply:
But I agree, it very easy to forget to add them in here!